gradle/libs.versions.toml

```diff
@@ -10,7 +10,7 @@ guava = "31.1-jre"
 lombok = "1.18.20"
 slf4j = "2.0.9"
 log4j = "2.22.0"
-jetty = "9.4.51.v20230217"
+jetty = "9.4.53.v20231009"
```

Comment:
Why did we update the jetty version?

Reply:
I can revert the version. I bumped the version to see if it can fix the JDK17 problem I met yesterday, seem unrelated.

Reply:
> I can revert the version.

Would you revert the change here? If no modifications are made later, I will approve it now.

Reply:
I will do it.